t/ui/26-jobs_restart.t

```diff
@@ -142,7 +142,7 @@ subtest 'restart job from info panel in test results' => sub {
     subtest 'successful restart' => sub {
         is($driver->get('/tests/99946'), 1, 'go to job 99946');
         update_last_job_id;
-        $driver->find_element('button > #restart-result')->click();
+        $driver->find_element('#restart-result')->click();
```

Comment:
You're changing it here. But the test wasn't failing before either. Please convince me why this should work now w/o other changes, or extend the test accordingly

Reply:
Added according tests for the redirection after clicking the advanced menu entries same like on the default action where it's already tested